hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -176,7 +176,7 @@ <h2 class="formH2" id="faq">Application FAQs</h2>
             <p class="faqAnswer">Your team number is just for your application so we can review your team together. It is not deciding and not needed during the event. You have until the event to choose a team and must submit to Devpost with the final team.</p>
 
             <p class="faqQuestion">I don’t have a team. What can I do?</p>
-            <p class="faqAnswer">All teams competing must have 3 or 4 members, but you do not need a full team in order to apply. We have a #team-formation channel on Slack as well as a team formation event scheduled before hacking starts at the event.</p>
+            <p class="faqAnswer">All teams competing must have 3 or 4 members, but you do not need a full team in order to apply. We have a #team-formation channel on {{ chat_room_name }} as well as a team formation event scheduled before hacking starts at the event.</p>
 
```

Comment:
I would probably rewrite this to not contain #team-formation either, since that's a slack thing. Some generic answer is fine, since the expectation is that people will rewrite these FAQ anyway